metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/SwaggerConfig.java

```diff
@@ -92,7 +92,8 @@ public Docket metadataApi() {
                         new Tag("Sample Entity", "Sample metadata"),
                         new Tag("Study Entity", "Study metadata"),
                         new Tag("Taxonomy Entity", "Taxonomy definitions"),
-                        new Tag("WebResource Entity", "Study resources that can be accessed through web protocols")
+                        new Tag("WebResource Entity", "Study resources that can be accessed through web protocols"),
+                        new Tag("TaxonomyTree Entity", "Taxonomy tree")
```

Comment:
If I remember correctly we decided to not use TaxonomyTree at all, storing everything directly in Taxonomy entity as well; why are there still references to TaxonomyTree then?

Reply:
sorry will remove that. i removed the entities but forgot to remove the references.